src/features/identityCheck/useIdentityCheckSteps.test.tsx

```diff
@@ -94,4 +101,22 @@ describe('useIdentityCheckSteps', () => {
 
     expect(steps[0].screens.includes('PhoneValidationTooManySMSSent')).toEqual(false)
   })
+
+  it.each`
+    enableNewIdentificationFlow | expectedUbbleFlow
+    ${true}                     | ${['SelectIDOrigin']}
+    ${false}                    | ${['IdentityCheckStart', 'IdentityCheckWebview', 'IdentityCheckEnd']}
+  `(
+    'should return $expectedUbbleFlow identity screen list if $enableNewIdentificationFlow enable',
```

Comment:
```suggestion
    'should return $expectedUbbleFlow identity screen list when enableNewIdentificationFlow is $enableNewIdentificationFlow',
```
Sinon pendant l'exécution du test ça aura cette tête là: "should return ["SelectIDOrigin"] identity screen list if **true** enable"